voorhoede/tudelft-spacefinder
# Add navigation button

# Changes

The share button is replaced with a navigation button. This navigation button links to the coordinates of the space on Google Maps.

- Replace the share button with the route button.
- Move styling of positioning of buttons to spaces view for cleaner CSS.
- Position space card above the navigation instead of slightly behind it.

# Associated issue

Resolves https://trello.com/c/R0PtYStb/84-make-sure-that-a-new-student-can-easily-navigate-to-a-location

# How to test

1. Open preview link on desktop and mobile.
2. Click on the route button; this should open an new tab with Google Maps.
3. On the map a marker must be shown of the location of the space.

# Checklist
